package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@inrupt/solid-react-components",
-  "version": "0.5.3",
+  "version": "0.5.4",
```

Comment:
The version change should be featured in its own commit, in order to be able to tag it.

Reply:
Yeah, good point - I'll try to remember to discuss that with Jack as we do the release!